### DISCUSSION TITLE
How do I point the plugin to a custom Jython environment?

### DISCUSSION THREAD

**Discussion by EricSalemi** [qualersoft/robotframework-gradle-plugin#55] [Q&A]
Hi,

I'm setting up some integration tests for my project and wanted to try out this plugin. Currently I can execute some dummy RF tests and my custom RF library written in pure Python is being used. However I don't know how I can import extra Python libraries into the environment. Is there a way to customize the provided Jython environment? Can I prepare my own Jython environment and hook it into this plugin?

**Comment by mathze** [comment]
Hi @EricSalemi,

that's a good question :) never tried it with python rf libs.
Under the hood the plugin just utilize the org.robotframework:robotframework library, which brings in the robotframework and jython stuff. So about the question about "customize the provided Jython environment" I would say: so far as I know, this isn't possible.
But about the part with "python RF library" that should be possible. As far as I understand the documentation right, you can use the `additionalPythonPaths` property for that. Just point it to the location of the libraries. The property is a `ConfigurableFileCollection` so you can add more than one location.

I hope I was able to help you, and thank you very much for trying out my plugin:).
